modules/oracle-xe/src/main/java/org/testcontainers/containers/OracleContainer.java

```diff
@@ -1,45 +1,40 @@
 package org.testcontainers.containers;
 
 import com.google.common.collect.Sets;
+import org.apache.commons.lang.StringUtils;
 import org.testcontainers.utility.DockerImageName;
-import org.testcontainers.utility.TestcontainersConfiguration;
 
+import java.util.Arrays;
+import java.util.List;
 import java.util.Set;
 import java.util.concurrent.Future;
 
-/**
- * @author gusohal
- */
-public class OracleContainer extends JdbcDatabaseContainer<OracleContainer> {
+public class OracleContainer<SELF extends OracleContainer<SELF>> extends JdbcDatabaseContainer<SELF> {
```

Comment:
no need for SELF-typing here:
```suggestion
public class OracleContainer extends JdbcDatabaseContainer< OracleContainer > {
```